backends/platform/android/events.cpp

```diff
@@ -373,6 +373,13 @@ static const Common::KeyCode jkeymap[] = {
 	Common::KEYCODE_PASTE // AKEYCODE_PASTE
 };
 
+void OSystem_Android::updateEventScale(uint32 w, uint32 h) {
+	if ((h != 0) && (w != 0)) {
+	    _eventScaleY = 100 * 480 / h;
```

Comment:
What are these magic numbers?

Reply:
This a bare copy from android3d backend.
In current code, the `_eventScaleX` and `_eventScaleY` are set to 100 and never touched again.
android3d has this magical function. I suppose 480 is related to good ol' 640x480 and 100 is to have a percentage but I don't know the rationale behind this.
This scale seems to be used by directional pads, trackballs and joysticks and I don't have any to test on my android.

Reply:
maybe add comments in code for now

Reply:
After some digging in codebase, this function did exist in the past and got removed by #1695.
I don't know how useful it was. Maybe @ccawley2011 knows better.
Removing for now. We can add it back later if we want to.

Reply:
indeed